core/widgets/SubmitButtonWidget.php

```diff
@@ -42,9 +42,31 @@ class SubmitButtonWidget extends Widget
     public $options = [];
 
     /**
+     * @since 1.0.24 First time this was introduced
+     *
      * @var activeForm Define activeForm context to use widget in context and only disable button when validation succeeded
      */
-    public $activeForm;
+    protected $activeForm;
```

Comment:
in general we store getter and setter as privat variables:

private $_activeForm